homeassistant/components/lcn/strings.json

```diff
@@ -26,7 +26,8 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
           "sk_num_tries": "Segment coupler scan attempts",
-          "dim_mode": "Dimming mode"
+          "dim_mode": "Dimming mode",
+          "acknowledge": "Request acknowledgement from modules"
```

Comment:
Should we use `data_description` to give more context to the user about what this means for them?

Reply:
I added the `data_description` to those two options.